packages/bezier-react/src/components/Forms/Inputs/TextField/TextField.test.tsx

```diff
@@ -285,4 +291,54 @@ describe('TextField', () => {
       })
     })
   })
+
+  describe('show remove button only when it is filled and focused/hovered', () => {
```

Comment:
> user-event allows you to describe a user interaction instead of a concrete event. It adds visibility and interactability checks along the way and manipulates the DOM just like a user interaction in the browser would. It factors in that the browser e.g. wouldn't let a user click a hidden element or type in a disabled text box.
This is [why you should use user-event](https://ph-fritsche.github.io/blog/post/why-userevent) to test interaction with your components.
> https://testing-library.com/docs/user-event/intro

userEvent를 사용하는 게 유저의 동작을 모사한다는 점에 있어서 더 좋을 거 같아요~!